### DISCUSSION TITLE
Reset a raid?

### DISCUSSION THREAD

**Discussion by conjections** [trickerer/Trinity-Bots#221] [Q&A]
What gm command resets a raid instances? Is there one?? I would assume there is. Somebody told me to use the one to unbind an instance, but that doesn't seem to reset it. I just get prompted that I can either rebind or leave the instance upon entry.

**Comment by trickerer** [comment]
1. Select self
2. `.ins unb all`
3. Disband group

**Reply by xBarselonx** [reply]
А для АС эта процедура сброса кд подойдет? Просто в АС проблема с этим.

**Reply by trickerer** [reply]
> А для АС эта процедура сброса кд подойдет? Просто в АС проблема с этим.

Не пробовал, но думаю что да

**Reply by xBarselonx** [reply]
Спасибо! Помог. Все работает.
